include/mender-client.h

```diff
@@ -54,6 +55,11 @@ typedef struct {
     uint32_t inventory_update_interval; /**< Inventory update interval, default is compile-time defined */
 #endif                                  /* CONFIG_MENDER_CLIENT_INVENTORY */
     bool recommissioning;               /**< Used to force creation of new authentication keys */
+    struct {
+        MenderAllocator   mender_malloc_func;
+        MenderReallocator mender_realloc_func;
+        MenderDeallocator free_func;
```

Comment:
Same here (i.e., no `mender_` suffix). Maybe this was the intention?

Reply:
No it wasn't. It was me using `sed s/free/mender_free/` multiple times, accidentally, and then trying to fix it. Good :eyes: !